other/upgrade.php

```diff
@@ -5049,53 +5056,20 @@ function MySQLConvertOldIp($targetTable, $oldCol, $newCol, $limit = 50000, $setS
 						ELSE NULL
 					END
 					WHERE ' . $oldCol . ' IN ({array_string:whereSet})',
-					$updates
+					array_merge($updates, $new_ips)
```

Comment:
to mutch stuff happen here in one line my pov

Reply:
Works fine.  Nothing wrong with using an array_merge when necessary.